templates/governing-board/working_group.html

```diff
@@ -0,0 +1,74 @@
+{% extends "page.html" %}
+{% block content -%}
+<article class="content">
+    <header>
+        <h1>{{ page.title }}</h1>
+    </header>
+
+    {% set wg_path = page.path | split(pat="/") | slice(end=-2) | join(sep="/") %}
+    {% set wg_path = wg_path ~ "/working_groups.toml" %}
+    {% set working_groups = load_data(path=wg_path, format="toml") %}
+
+    {% for wg in working_groups.working_groups %}
+    {% if page.title is not starting_with(wg.name) %}
+    {% continue %}
+    {% endif %}
+
+    {% if wg.committee == "Community Committee" %}
+    {% set committee_id = "community" %}
+    {% elif wg.committee == "Governance Committee" %}
+    {% set committee_id = "governance" %}
+    {% elif wg.committee == "Trust & Safety Committee" %}
+    {% set committee_id = "trust-and-safety" %}
+    {% elif wg.committee == "Finance & Fundraising Committee" %}
+    {% set committee_id = "finance" %}
+    {% endif %}
+
+    <section class="working_group">
+        <div class="description">
+            <h2 id="charter">Charter</h2>
+            <blockquote>
```

Comment:
Seems like the left border currently is only happening on `article.post blockquote` so maybe fix css to work here too?